about.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="icon" href="images/logo-icon.png">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha3/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-KK94CHFLLe+nY2dmCWGMq91rCGa5gtU4mk92HdvYe+M/SXH301p5ILy+dN9+nJOZ" crossorigin="anonymous">
+    <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <title>Techrotics</title>
+</head>
+
+<body>
+    <header class="about">
+        <nav class="navbar navbar-expand-lg navbar-expand-md n">
+            <div class="container-fluid mx-lg-5 mx-md-5 mx-0">
+                <button class="navbar-toggler focus-ring focus-ring-light m-2 m-lg-0 m-md-0 " id="toggle-icon"
+                    type="button" data-bs-toggle="collapse" data-bs-target="#navbarTogglerDemo01"
+                    aria-controls="navbarTogglerDemo01" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon" id="icon"></span>
+                    <span id="cancel" class="pb-1 "><img src="images/cancel.png" alt=""></span>
+                </button>
+
+                <div class="collapse navbar-collapse" id="navbarTogglerDemo01">
+                    <a class="navbar-brand " href="index.html"><img src="images/logo.png" height="40px"
+                            class="mt-3 mx-2 mt-lg-0 mt-md-0" alt="logo"></a>
+                    <ul class="navbar-nav  mb-2 mb-lg-0 mx-2 mt-md-2 mt-lg-1 ms-auto">
+                        <li class="nav-item mx-3">
+                            <a class="nav-link " aria-current="page" href="index.html">Home</a>
+                        </li>
+                        <li class="nav-item mx-3">
+                            <a class="nav-link active" href="about.html">About</a>
+                        </li>
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done it is so far kindly implement the [previous reviewer's suggestion](https://github.com/batoolfatima2135/Capstone/pull/1#discussion_r1202385795) also `ABOUT` page 🙏🏼 

![image](https://github.com/batoolfatima2135/Capstone/assets/63915024/dc717c17-c19d-4fe0-aa3a-f9adbd22d9e5)